### PR TITLE
feat: Added Source Link support.

### LINKED ISSUE #7: Add Source Link support
Both **OperationResults** and **OperationResults.AspNetCore** projects must support Source Link, as described here: https://devblogs.microsoft.com/dotnet/producing-packages-with-source-link.

At https://github.com/ErikEJ/SqlClientExtensions/blob/main/src/SqlClientExtensions/SqlClientExtensions.csproj you can find a real example of a library with Source Link enabled.

### PR DESCRIPTION
**Pull request by Defkon1**
feat: Added Source Link support.
close #7

### CODE REVIEW

File: src/OperationResults.AspNetCore/OperationResults.AspNetCore.csproj [hunk 1]
```diff
@@ -30,6 +37,10 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="OperationResultTools" Version="1.0.3" />
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
```
Comment (marcominerva) [inline src/OperationResults.AspNetCore/OperationResults.AspNetCore.csproj:40]: NuGet references should be in alphabetic order.

File: src/OperationResults/OperationResults.csproj [hunk 2]
```diff
@@ -18,13 +18,24 @@
...
+    <PropertyGroup Condition="'$(GITHUB_ACTIONS)' == 'true'">
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="Nerdbank.GitVersioning" Version="3.5.109">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
```
Comment (marcominerva) [inline src/OperationResults/OperationResults.csproj:35]: NuGet references should be in alphabetic order.

**Review by marcominerva — CHANGES_REQUESTED** [review]
The Pull request must be made to the `develop` branch.

File: src/OperationResults.AspNetCore/OperationResults.AspNetCore.csproj (hunk 1 above)
Comment (Defkon1) [reply src/OperationResults.AspNetCore/OperationResults.AspNetCore.csproj]: Fixed with 5f25a941423c9bed94670b0c42aad99702351d08

File: src/OperationResults/OperationResults.csproj (hunk 2 above)
Comment (Defkon1) [reply src/OperationResults/OperationResults.csproj]: Fixed with 5f25a941423c9bed94670b0c42aad99702351d08

**Comment by Defkon1** [comment]
Base branch changed and dependencies reordered ;-)

**Review by marcominerva — APPROVED** [review]

**Pull request merged**